analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -246,6 +254,23 @@ public void perform() {
                 cache.addGAV(project, current);
             }
 
+            AppliedPlugin ap = project.getPluginManager().findPlugin(DOKKA);
+            if (ap != null) {
+                logger.debug("Plugin {} has been applied to {}", ap.getId(), project.getName());
+
+                @SuppressWarnings("rawtypes")
+                Plugin p = ((DefaultPluginManager) project.getPluginManager()).getPluginContainer().findPlugin(DOKKA);
+                String path = p.getClass().getProtectionDomain().getCodeSource().getLocation().getPath();
+                Matcher m = Pattern.compile(".*-([\\d.]+[a-zA-Z]*)\\.jar").matcher(path);
+                if (m.find()) {
```

Comment:
I think you want `matches()` instead of `find()` since `.jar` should be the very end of the string (equivalent to `\\.jar$`).

Reply:
Yeah that sounds reasonable, will change, thanks.